src/library/library.cpp

```diff
@@ -47,10 +47,15 @@ Library::Library(QObject* parent, UserSettingsPointer pConfig,
                  RecordingManager* pRecordingManager) :
         m_pConfig(pConfig),
         m_pSidebarModel(new SidebarModel(parent)),
-        m_pTrackCollection(new TrackCollection(pConfig, TrackCollection::kDefaultSchemaFile)),
+        m_pTrackCollection(new TrackCollection(pConfig)),
         m_pLibraryControl(new LibraryControl(this)),
         m_pRecordingManager(pRecordingManager),
         m_scanner(m_pTrackCollection, pConfig) {
+    if (!m_pTrackCollection->initDatabaseSchema()) {
```

Comment:
I've seen your bug report ;) But I don't have a strategy that works for all tests, this one is an exception.